src/main.c

```diff
@@ -97,6 +100,68 @@ unsigned long AuxFragBufOffset = 0;
 DOT11_EXTSTA_RECV_CONTEXT PacketMetadata;
 BOOLEAN AddMetadata = FALSE;
 
+typedef struct _NDIS_NET_BUFFER_LIST_8021Q_INFO
+{
```

Comment:
Look at existing structs- the brace goes on the same line as the "struct" keyword

Reply:
Same for unions

Reply:
I've pushed a fix for this and other remaining brace style deviations.